File: uva/272_TEXQuotes.cpp
#include <bits/stdc++.h>
#include <string>
using namespace std;

int main()
{
	int count=1,idx;
	string kalimat,kalimatBaru,a;
	while(true)
	{
		kalimatBaru="";
		getline(cin,kalimat);
		if(cin.eof()) break;
		for(string::iterator i=kalimat.begin(); i!=kalimat.end(); ++i)
	{
		a = *i;
		if(a=="\"" and count%2!=0)
		{
			idx = kalimat.find(a);
			kalimatBaru += "``";
			count +=1;
		}
		else if(a=="\"" and count%2==0)
		{
			idx = kalimat.find(a);
			kalimatBaru +=	"''";		
			count +=1;	
		}
		else
		kalimatBaru += a;
	}
	cout<<kalimatBaru<<"\n";
	}
	return 0;
}

File: uva/272_TEXQuotes.py
count = 1
while True:
    try:
        kalimat = input()
        for i in kalimat:
            if i == "\"" and count % 2 != 0:
                idx = kalimat.index(i)
                kalimat = kalimat[:idx]+"``"+kalimat[idx+1:]
                count += 1
            elif i =="\"" and count % 2 == 0:
                idx = kalimat.index(i)
                kalimat = kalimat[:idx]+"\'\'"+kalimat[idx+1:]
                count += 1
        print(kalimat)
    except EOFError:
        break
